### ISSUE TITLE
Support of self-hosted Langfuse

### ISSUE THREAD

**Issue by yassineselmi** [logspace-ai/langflow#942]
It appears that the existing version does not provide compatibility with self-hosted Langfuse, even though the ability to set the environment variable LANGFLOW_LANGFUSE_HOST is stated in the documentation.

1. Deploy Langfuse locally
2. Set the env variables: `LANGFLOW_LANGFUSE_HOST`, `LANGFLOW_LANGFUSE_SECRET_KEY`, and `LANGFLOW_LANGFUSE_PUBLIC_KEY`
3. Start/Restart Langflow
4. Run any model
5. See the following errors:

```
[14:37:30] ERROR    [14:37:30] - ERROR - 1 validation error for    client.py:445
...
---> 98 │   │   │   raise UnauthorizedError( │                    │       type: ignore
```

**Proposed Fix**

Actually, Langflow is not taking in consideration the `LANGFLOW_LANGFUSE_HOST`  when instantiating the Langfuse object. 

In [plugins.langfuse.py](https://github.com/logspace-ai/langflow/blob/v0.5.0a1/src/backend/langflow/services/plugins/langfuse.py#L25): 

```python
cls._instance = Langfuse(
      public_key=settings.LANGFUSE_PUBLIC_KEY,
      secret_key=settings.LANGFUSE_SECRET_KEY,
  )
```

must be modified to:

```python
cls._instance = Langfuse(
      host=settings.LANGFUSE_HOST or "https://cloud.langfuse.com",
      public_key=settings.LANGFUSE_PUBLIC_KEY,
      secret_key=settings.LANGFUSE_SECRET_KEY,
 )
```

I will submit a PR with the proposed fix.

**Comment by yamonkjd** [comment]
Surprisingly, I opened the issues tab to make the same suggestion. I actually used the cloud version of Langfuse and found it to be a great help when creating flows, so I think this feature is absolutely necessary. Going a little further, it would be incredibly efficient if it were displayed together with the lang flow log output in debug mode.

**Comment by yassineselmi** [comment]
I see that it has been fixed in #948  and release in v0.4.21.

Thank you.

**Comment by yassineselmi** [comment]
fixed in #948 and released (v0.4.21).

Thank you.
